org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/entities/EntitiesCompletionExtensionsTest.java

```diff
@@ -78,6 +78,24 @@ public void inside() throws BadLocationException {
 				c("&mdash;", "&mdash;", r(5, 2, 5, 14), "&mdash;"));
 	}
 
+	@Test
+	public void underscoreEntityName() throws BadLocationException {
+		// &foo_b|
+		String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + //
+				"<!DOCTYPE article [\r\n" + //

```

Comment:
shouldn't it be `DOCTYPE  root` to be more consistent?

Reply:
fixed